lib/widget/game.dart

```diff
@@ -175,6 +179,59 @@ class GameState extends State<Game> {
       ));
     }
 
+    // show the balances the accountant knows
+    if (me.role == ACCOUNTANT.roleId) {
+      // the accountant can reveal a balance per completed heist up to a maximum of half the number
+      // of players rounded down
+      int completedHeists =
+          getHeists(_store.state).where((heist) => heist.completedAt != null).length;
+      int numPlayers = getRoom(_store.state).numPlayers;
+      int maxBalances = min(completedHeists, (numPlayers / 2).floor());
+      basicTiles.add(
+        new ListTile(
+          title: new Text(
+            'You can also see the balance of up to $maxBalances people:',
+            style: infoTextStyle,
+          ),
+        ),
+      );
+
+      Set<String> visibleToAccountant = getRoom(_store.state).visibleToAccountant;
+
+      visibleToAccountant?.forEach((String playerId) {
+        Player player = getPlayerById(_store.state, playerId);
+        basicTiles.add(
+          new ListTile(
+            title: new Text(
+              '${player.name}: ${calculateBalanceFromStore(_store, player)}',
```

Comment:
This (the player's balance) won't update while you are looking at it, as it's not part of the view model.

An easy way might be to include the rounds in the view model (even though you aren't referring to them).